ampoule/pool.py

```diff
@@ -241,7 +242,13 @@ def cancelCall(call):
             cancelCall(timeoutCall)
             cancelCall(deadlineCall)
             self.busy.discard(child)
-            if not die:
+
+            already_dead = (
```

Comment:
This should be `alreadyDead` since ampoule uses Twisted rather than PEP8 variable naming convention.  (But given the presence of `is_error` right here, it's clearly inconsistent and I can see why that would not be obvious.)

Reply:
Fixed in 07b6bce